sdk/src/container.js

```diff
@@ -15,16 +15,16 @@ F2.extend('', (function() {
    * @method inArray
    * @param {object} value The value to search for
    * @param {Array} array The array to search
-   * @return {bool} True if the item is in the array
+   * @return {int} index of the value in the array, -1 if value not found
```

Comment:
Why did this need to change from returning a `bool` to an `int`?

Reply:
Hi @brianbaker,  Looks like earlier we were using jQuery.inArray() which returns the index of the item, and also in the code
we are checking (https://github.com/OpenF2/F2/blob/master/sdk/src/container.js#L346) against the integer so condition to get styles url were failing and styles were not loading. So, I changed this to return integer as we are checking instead of modify all the conditions.